### DISCUSSION TITLE
Alternate songs added by different users

### DISCUSSION THREAD

**Discussion by AlvaroMartinezB** [codetheweb/muse#995] [Ideas]
Would be cool if the Queue could alternate between songs added by different users. Example: User 1 adds 10 songs. User 2 then adds 1, it would be cool if this song now became number 2 in the queue. That way if everyone spams the queue with a disproportionate number of songs, we will listen to one song from each user at a time. 

Hope my thought came through. I know you can shuffle the queue, but if someone added 40 songs, and the other person added 2 the odds of listening to those 2 songs soon are slim.

**Comment by codetheweb** [comment]
Open to a PR for this. I think maybe the best way to implement it would be to have this be a configurable setting per guild (`queue-ordering: fairly-weighted`?).
